packages/ui/src/components/atoms/Typography.tsx

```diff
@@ -29,8 +29,9 @@ export type TypographyColor =
   | 'warn'
   | 'list'
   | 'black'
+  | 'info'
   | 'disabled'
-  | 'info';
+  | 'red';
```

Comment:
error is already present which does the same thing

Reply:
removed